.github/composites/install/action.yaml

```diff
@@ -67,7 +67,7 @@ runs:
         if [ -d ".venv" ]; then
           mv .venv .venv-old
         fi
-        python -m venv .ven
```

Comment:
this is drive by ci pipeline fix, re anything changing a lock file will fail here, extant ci install is cached by lock hash